### PR TITLE
add message inline to thread admin

### PR DESCRIPTION
**Pull request by ilikerobots**
Adds a thread's messages as inline in Thread admin

### CODE REVIEW

**Comment by amandasavluchinske** [comment]
Hey @ilikerobots , thanks for your contribution!

Just wanted to give you a heads up that @fjsj is on PTO right now and will be coming back on Aug 1st. He'll have a look at your PR once he's back.

File: django_ai_assistant/admin.py [hunk 1]
```diff
@@ -1,19 +1,70 @@
...
+        return False
+
+
 @admin.register(Thread)
 class ThreadAdmin(admin.ModelAdmin):
     list_display = ("name", "created_at", "created_by", "updated_at")
     search_fields = ("name",)
     list_filter = ("created_at", "updated_at")
     raw_id_fields = ("created_by",)
+    inlines: ClassVar[InlineModelAdmin] = [MessageInline]
```
Comment (filipeximenes) [inline django_ai_assistant/admin.py:62]: I believe the correct type annotation here is `List[Type[InlineModelAdmin]]`

File: django_ai_assistant/admin.py [hunk 2]
```diff
@@ -1,19 +1,70 @@
...
+    show_change_link = True
+
+    def pk(self, obj):
+        display_text = "<a href={}>{}</a>".format(
+            reverse(
+                f"admin:{Message._meta.app_label}_{Message._meta.model_name}_change", args=(obj.pk,)
+            ),
+            obj.pk,
+        )
+        if display_text:
```
Comment (filipeximenes) [inline django_ai_assistant/admin.py:36]: Please correct if I'm wrong, but I believe it's not possible for this `if` statement to resolve ever be falsy, I think we can remove it and return directly

File: django_ai_assistant/admin.py [hunk 3]
```diff
@@ -1,19 +1,70 @@
...
     search_fields = ("name",)
     list_filter = ("created_at", "updated_at")
     raw_id_fields = ("created_by",)
+    inlines: ClassVar[InlineModelAdmin] = [MessageInline]
 
 
 @admin.register(Message)
 class MessageAdmin(admin.ModelAdmin):
-    list_display = ("id", "thread", "created_at", "created_at")
+    list_display = ("id", "thread", "created_at")
```
Comment (filipeximenes) [inline django_ai_assistant/admin.py:67]: Good catch!

**Review by filipeximenes — CHANGES_REQUESTED** [review]

**Comment by filipeximenes** [comment]
Thanks for your contributions @ilikerobots! I've taken the liberty to address the issues and merge your changes (#159).

**Pull request closed without merging**